platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerView.swift

```diff
@@ -25,6 +25,7 @@ public struct WysiwygComposerView: UIViewRepresentable {
     public var replaceText: (NSAttributedString, NSRange, String) -> Bool
     public var select: (NSAttributedString, NSRange) -> Void
     public var didUpdateText: (UITextView) -> Void
+    private var textColor: UIColor = .label
```

Comment:
```suggestion
    private var textColor = Color(.label)
```

Should probably also make the property a `Color` I think properties using in swiftui should be value types or you can get weirdness.

Reply:
Yeah I agree, I can even use Color.primary since is the SwiftUI equivalent of UIColor.label